app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorEffectHandler.kt

```diff
@@ -4,11 +4,17 @@ import com.f2prateek.rx.preferences2.Preference
 import com.spotify.mobius.rx2.RxMobius
 import com.squareup.inject.assisted.Assisted
 import com.squareup.inject.assisted.AssistedInject
+import io.reactivex.Observable
 import io.reactivex.ObservableTransformer
+import io.reactivex.rxkotlin.Observables
 import org.simple.clinic.sync.LastSyncedState
+import org.simple.clinic.util.UtcClock
+import org.threeten.bp.Instant
 
 class SyncIndicatorEffectHandler @AssistedInject constructor(
     private val lastSyncedState: Preference<LastSyncedState>,
+    private val utcClock: UtcClock,
+    private val syncIndicatorConfig: Observable<SyncIndicatorConfig>,
```

Comment:
This change will be made as part of the cleanup. This required changing tests considerably and it wasn't possible during the migration itself.

Reply:
I personally prefer doing this as part of test stabilization since it simplifies the migration effort greatly. I guess it's something I forgot to add. 😓